Button/button.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Bricks component library</title>
```

Comment:
Instead of giving the CL name as title, you should be more specific, such as 
`Buttons | Bricks component library`
since what the user will see are the first words in the browser tab.